recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
```

Comment:
Or maybe conda-build should just be updated to automatically convert git_url and git_rev into the tarball url